terraform/environments/electronic-monitoring-data/s3_main.tf

```diff
@@ -79,4 +79,75 @@ resource "aws_s3_bucket_notification" "send_metadata_to_ap" {
   }
 
   depends_on = [aws_lambda_permission.send_metadata_to_ap]
-}
\ No newline at end of file
+}
+
+# ----------------------------------
+# Athena Query result storage bucket
+# ----------------------------------
+module "athena-s3-bucket" {
+  source = "github.com/ministryofjustice/modernisation-platform-terraform-s3-bucket?ref=cadab51"
+
+  bucket_name        = join("-", ["aws-athena-query-results", local.env_account_id, local.region])
+  versioning_enabled = true
+
+  # to disable ACLs in preference of BucketOwnership controls as per https://aws.amazon.com/blogs/aws/heads-up-amazon-s3-security-changes-are-coming-in-april-of-2023/ set:
+  ownership_controls = "BucketOwnerEnforced"
+  acl                = "private"
+
+  # Refer to the below section "Replication" before enabling replication
+  replication_enabled = false
+  # Below variable and providers configuration is only relevant if 'replication_enabled' is set to true
+  # replication_region                       = "eu-west-2"
+  providers = {
+    # Here we use the default provider Region for replication. Destination buckets can be within the same Region as the
+    # source bucket. On the other hand, if you need to enable cross-region replication, please contact the Modernisation
+    # Platform team to add a new provider for the additional Region.
+    # Leave this provider block in even if you are not using replication
+    aws.bucket-replication = aws
+  }
+
+  lifecycle_rule = [
+    {
+      id      = "main"
+      enabled = "Enabled"
+      prefix  = ""
+
+      tags = {
+        rule      = "log"
+        autoclean = "true"
+      }
+
+      transition = [
+        {
+          days          = 14
+          storage_class = "STANDARD_IA"
+          }, {
+          days          = 365
+          storage_class = "GLACIER"
+        }
+      ]
+
+      expiration = {
+        days = 14
+      }
+
+      noncurrent_version_transition = [
+        {
+          days          = 14
+          storage_class = "STANDARD_IA"
+          }, {
+          days          = 365
+          storage_class = "GLACIER"
+        }
+      ]
+
+      noncurrent_version_expiration = {
+        days = 14
+      }
+    }
+  ]
+
+  version_enabled = true
```

Comment:
Is this a variable for the module? We couldn't see it

Reply:
oops, removed it, as it's already at the top.